services/docker/docker-version.service.js

```diff
@@ -3,14 +3,14 @@
 const Joi = require('@hapi/joi')
 const { nonNegativeInteger } = require('../validators')
 const { latest, renderVersionBadge } = require('../version')
+const { NotFound, InvalidResponse } = require('..')
+const { BaseJsonService } = require('..')
```

Comment:
Same thought as the other docker service. Not sure whether prettier will bark this not being multilined
```suggestion
const { BaseJsonService, NotFound, InvalidResponse } = require('..')
```